src/components/Modal/ConsentModal.tsx

```diff
@@ -1,35 +1,29 @@
 import { Dialog } from "@headlessui/react"
-import { FC, useEffect, useState } from "react"
+import { FC } from "react"
+
+import { useClientReady } from "@/hooks"
 
 import Button from "@/components/Button"
 import PurpleModal, {
   PurpleModalContent,
   PurpleModalHeader,
-} from "@/components/Modal/PurpleModal"
-
-import useConsentStorage from "@/store/consent"
+} from "@/components/Modal/lib/PurpleModal"
 
-export const Consent: FC = () => {
-  const consent = useConsentStorage((state) => state.consent)
-  const understandDisclaimer = useConsentStorage(
-    (state) => state.understandDisclaimer
-  )
-  const [isOpen, setIsOpen] = useState(false)
```

Comment:
No state management was ever required for this. We can fully rely on the store to manage this component's state.